.github/workflows/dependabot-auto-approve.yml

```diff
@@ -0,0 +1,22 @@
+name: Dependabot auto-approve
+on: pull_request
+
+permissions:
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v1
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+      - name: Approve a PR
+        if: ${{(contains(steps.metadata.outputs.dependency-names, 'eslint') || contains(steps.metadata.outputs.dependency-names, '@types')) && (steps.metadata.outputs.update-type == 'version-update:semver-patch' || steps.metadata.outputs.update-type == 'version-update:semver-minor')}}
```

Comment:
Is it safe to use `contains`? Have you considered using [startsWith](https://docs.github.com/en/actions/learn-github-actions/expressions#startswith) for a more strict check?

Reply:
We could make it more precise but need to grow it then. Currently it captures these dependencies:

<img width="490" alt="Screenshot 2023-06-28 at 10 18 07" src="https://github.com/filecoin-station/desktop/assets/10247/ac867b0d-be9e-404b-bcd3-0c5837317e9f">

Wdyt?

Reply:
I would create two `startsWith` conditions: `@typescript-eslint` and `eslint`.

Having said that, I think the likelihood of a production dependency containing `eslint` in the name is low enough to make `contains` a good enough solution.

Reply:
Good point! I'll refactor it. Better safe than sorry